addons/test_discuss_full/tests/test_performance.py

```diff
@@ -440,6 +440,7 @@ def _get_init_store_data_result(self):
                 "self_partner": self.users[0].partner_id.id,
                 "settings": {
                     "channel_notifications": False,
+                    'color_scheme': 'system',
```

Comment:
Hello, this will fail in community-only run, this should be added in the enterprise override.